src/components/reusable/header/MainHeader.tsx

```diff
@@ -85,11 +90,13 @@ export default function MainHeader({
           onClick={viewNotifications}
           className={route === ROUTES.notifications ? 'active' : ''}
         >
-          <NotificationsIcon />
+          <Badge badgeContent={unreadNotifications?.count} color="primary" max={99}>
```

Comment:
same thing here, I think the badge should be rendered conditionally, if we don't have unreadNotifications, than we shouldn't render it

Reply:
fixed